### DISCUSSION TITLE
External Procedure Definitions

### DISCUSSION THREAD

**Discussion by gcraig88keys** [codefori/vscode-ibmi#2129] [IBM i]
When I hover over procedures that are defined in external code (/copy or /include) I'm not seeing the definition window like I do with those defined internally.    

Here's an example of what I'm expecting, but this is a procedure defined internally in this source member.
![image](https://github.com/codefori/vscode-ibmi/assets/89139309/c5a345de-99a1-41d5-b957-4b60ec372d42)

**Comment by worksofliam** [comment]
When you hover over the copy it come from, does it say in the hover that it was found?

**Reply by gcraig88keys** [reply]
No it doesn't.  The library list is set right so it should be able to find it.

**Reply by worksofliam** [reply]
Does it say 'not found' then? Or nothing at all?

**Reply by gcraig88keys** [reply]
It doesn't say anything.  The editor doesn't appear to do anything when I hover over the /copy or the actual procedure call. 

![image](https://github.com/codefori/vscode-ibmi/assets/89139309/2991f94e-d726-429d-904e-1e53b8d8c4dd)

![image](https://github.com/codefori/vscode-ibmi/assets/89139309/3c36c8ea-b29e-4e6c-af30-11b7958c3635)
